### ISSUE TITLE
Support for Parcel Bundler

### ISSUE THREAD

**Issue by babsonmatt** [ant-design/ant-design#8716]
### What problem does this feature solve?
I'd like to use parcel instead of webpack however doing so means I have to import the entire antd.css and can't benefit from babel-plugin-import to selectively import the required css.

### What does the proposed API look like?
https://parceljs.org/plugins.html

**Comment by Adherentman** [comment]
I hope to use parcel can use antd🤓

**Comment by shepherdwind** [comment]
￼￼￼￼I add antd to dependencies, and then build it with Parcel, The build cost more than 35s and the build result is very big 13M.

![image](https://user-images.githubusercontent.com/452899/34325242-71a1560c-e8c7-11e7-90ff-9863b52b5739.png)

It seems that babel-plugin-import not working, and all antd component all bundled together.

**Comment by wenpengfei** [comment]
The same requirement. Hope to add a pracel-plugin like `babel-plugin-import`

**Comment by afc163** [comment]
Parcel support babel presets and plugins: https://en.parceljs.org/transforms.html

Here is a demo repository: https://github.com/ant-design/parcel-antd

**Comment by babsonmatt** [comment]
@afc163 I'm not sure if this is unrelated, but if you try to add an Input.Search to the DOM instead of button, does everything work properly?  I get something like this:


![image](https://user-images.githubusercontent.com/853220/34472080-11b5bd76-ef27-11e7-81bf-6499e68f303c.png)


```
import React from 'react';
import ReactDOM from 'react-dom';
import { Input } from 'antd';

const Search = Input.Search;

ReactDOM.render(
  <Search placeholder="input search text" enterButton="Search" size="large" />,
  document.getElementById('mountNode'),
);
```

**Comment by afc163** [comment]
@babsonmatt It is a bug fixed in master: #8700

**Comment by babsonmatt** [comment]
@afc163 thank you!
